translations/town.tex/pl.po

```diff
@@ -80,7 +80,21 @@ msgid ""
 "Some Building Tiles are double-sided, and may later be upgraded and flipped to represent two different buildings at the same time. Such upgrades must be \\textbf{Built in order}.\\par\n"
 "\n"
 msgstr ""
-
+"\\hypertarget{Miasto}{Każda}\\index{Miasto} frakcja ma własne miasto, które znajduje się na środku swojego kafla początkowego.\n"
+"Miasto jest twoją najważniejszą lokacją - wiele scenariuszy może się \\hyperlink{End}{zakończyć}, jeśli \\hyperlink{Categories}{oflaguje} je bohater wroga.\\par\n"
+"Stan twojego miasta i frakcji są ukazane na planszetce miasta.\n"
```

Comment:
może to tylko ja, ale ukazane brzmi jak gdyby było set in stone, nie lepiej prościej - widoczne / przedstawione?
+ wielkie litery, Plansza Miasta
```suggestion
"Stan twojego Miasta i Frakcji są ukazane na Planszy Miasta.\n"
```

Reply:
W oryginale jest "planszetka", więc tego bym się trzymał.